python/lsst/cp/verify/verifyStats.py

```diff
@@ -0,0 +1,510 @@
+            The exposure to measure.
+        keywordDict : `dict` [`str`, `str`]
+            A dictionary of keys to use in the output results, with
+            values the string name associated with the
+            `lsst.afw.math.statistics.Property` to measure.
+        statControl : `lsst.afw.math.StatControl`
+            Statistics control object with the parameters defined by
+            the config.
+
+        Returns
+        -------
+        ampStats : `dict` [`str`, `dict` [`str`, scalar]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the statistics measured and their values.
+        """
+
+        ampStats = {}
+
+        # Convert the string names in the keywordDict to the afwMath values.
+        # The statisticToRun is then the bitwise-or of that set.
+        statisticToRun = 0
+        statAccessor = {}
+        for k, v in keywordDict.items():
+            statValue = afwMath.stringToStatisticsProperty(v)
+            statisticToRun |= statValue
+            statAccessor[k] = statValue
+
+        # Measure stats on all amplifiers.
+        for ampIdx, amp in enumerate(exposure.getDetector()):
+            ampName = amp.getName()
+            theseStats = {}
+            ampExp = exposure.Factory(exposure, amp.getBBox())
+            stats = afwMath.makeStatistics(ampExp.getMaskedImage(), statisticToRun, statsControl)
+
+            for k, v in statAccessor.items():
+                theseStats[k] = stats.getValue(v)
+            ampStats[ampName] = theseStats
+
+        return ampStats
+
+    def unmaskedImageStats(self, exposure, statControl):
+        """Measure amplifier level statistics on the data read from disk,
+        ignoring all mask planes.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            The exposure to measure.
+        statControl : `lsst.afw.math.StatControl`
+            Statistics control object with the parameters defined by
+            the config.
+
+        Returns
+        -------
+        outputStats : `dict` [`str`, `dict` [`str`, scalar]]
+            A dictionary indexed by the amplifier name, containing
+            dictionaries of the statistics measured and their values.
+
+        """
+        noMaskStatsControl = afwMath.StatisticsControl(self.config.numSigmaClip,
+                                                       self.config.clipMaxIter,
+                                                       0x0)
+        return self.amplifierStats(exposure, self.config.unmaskedImageStatKeywords, noMaskStatsControl)
+
+    def normalizedImageStats(self, exposure, statControl):
+        """Measure amplifier level statistics on the data read from disk,
+        after dividing by the exposure time.
+
+        Parameters
+        ----------
+        exposure : `lsst.afw.image.Exposure`
+            The exposure to measure.
+        statControl : `lsst.afw.math.StatControl`
```

Comment:
`lsst.afw.math.StatisticsControl`

And it might be worth just calling the parameter `statisticsControl` for consistency?